cloudslang-compiler/src/test/resources/task-args-one-liner/flow_3.yaml

```diff
@@ -19,8 +19,8 @@ flow:
     - CheckWeather:
         do:
           ops.check_Weather: >
-             city   =       city_name,
-              country=str('Wonderland')
+             city   =       ${ city_name },
+              country=${ 'Wonderland' }
```

Comment:
I think that Wonderland should remain a string to keep the original functionality